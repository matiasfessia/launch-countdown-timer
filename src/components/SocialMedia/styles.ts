import { css } from '@emotion/react';

export const styles = {
  container: css`
    display: flex;
  `,
  iconLink: css`
    margin: 5px 15px;
  `,
  icon: css`
    &:hover {
      // filter generated by https://pranjaldubey.com/css-filter-generator/
      filter: invert(50%) sepia(50%) saturate(3858%) hue-rotate(315deg) brightness(88%) contrast(85%);
    }
  `
};
