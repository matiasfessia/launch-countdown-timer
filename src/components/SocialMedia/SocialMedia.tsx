/** @jsxImportSource @emotion/react */
import { styles } from './styles';
import iconFacebook from '../../assets/images/icon-facebook.svg';
import iconPinterest from '../../assets/images/icon-pinterest.svg';
import iconInstagram from '../../assets/images/icon-instagram.svg';

export const SocialMedia = () => (
  <div css={styles.container}>
    <a css={styles.iconLink} href="https://www.facebook.com/">
      <img css={styles.icon} src={iconFacebook} alt="facebook icon" />
    </a>
    <a css={styles.iconLink} href="https://www.pinterest.com/">
      <img css={styles.icon} src={iconPinterest} alt="pinterest icon" />
    </a>
    <a css={styles.iconLink} href="http://instagram.com/">
      <img css={styles.icon} src={iconInstagram} alt="instagram icon" />
    </a>
  </div>
);

export default SocialMedia;
