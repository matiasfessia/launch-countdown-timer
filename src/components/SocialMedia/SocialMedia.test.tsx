import { render } from '@testing-library/react';
import { SocialMedia } from './SocialMedia';

it('should renders correctly', () => {
  const { container } = render(<SocialMedia />);
  expect(container).toMatchSnapshot();
});
