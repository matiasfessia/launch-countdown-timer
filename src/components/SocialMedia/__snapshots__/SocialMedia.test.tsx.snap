// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<div>
  <div
    class="css-20wlgm-SocialMedia"
  >
    <a
      class="css-1eiqcch-SocialMedia"
      href="https://www.facebook.com/"
    >
      <img
        alt="facebook icon"
        class="css-1idlvul-SocialMedia"
        src="icon-facebook.svg"
      />
    </a>
    <a
      class="css-1eiqcch-SocialMedia"
      href="https://www.pinterest.com/"
    >
      <img
        alt="pinterest icon"
        class="css-1idlvul-SocialMedia"
        src="icon-pinterest.svg"
      />
    </a>
    <a
      class="css-1eiqcch-SocialMedia"
      href="http://instagram.com/"
    >
      <img
        alt="instagram icon"
        class="css-1idlvul-SocialMedia"
        src="icon-instagram.svg"
      />
    </a>
  </div>
</div>
`;
